packages/theme/src/components/alert.ts

```diff
@@ -13,7 +13,7 @@ const alertBodyCva = cva('flex gap-1 h-fit flex-1', {
   },
 })
 
-const alertCva = cva(['flex rounded-lg relative gap-1'], {
+const alertCva = cva(['flex rounded-lg relative gap-1 pointer-events-auto'], {
```

Comment:
Any reason for adding `pointer-events-auto`?

Reply:
use pointer-events-none in the viewport to allow click through it but it make the toast unclickable
so i set the pointer-event-auto to prevent that

Reply:
But can we pass the class in the alert used in the toast component?